app-frontend/src/app/pages/library/scenes/detail/detail.html

```diff
@@ -10,7 +10,9 @@ <h1 class="h3 page-title">
     <rf-map-container class="map-preview-container"
                       map-id="scene"
                       options="$ctrl.mapOptions"></rf-map-container>
-    <rf-scene-detail scene="$ctrl.scene"></rf-scene-detail>
+    <rf-scene-detail scene="$ctrl.scene"
```

Comment:
@designmatty I'll leave this to you, since that's purely styling and doesn't affect functionality